### PR TITLE
[Chips] Support Dynamic Type and show it in example.

### PR DESCRIPTION
**Pull request by romoore**
**Entry Chips Before**
chips-input-small-before

**Entry Chips After**
chips-input-small-after

**Chips Small (After)**
chips-small-after

**Chips Large (After)**
chips-large-after

### CODE REVIEW

**Comment by yarneo** [comment]
ht_tortilla_chip_2_dm_120621_wblog

File: components/Chips/tests/unit/ChipsTests.m [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  brightness = MIN(1, MAX(0, brightness - percent));
+
+  return [UIColor colorWithHue:hue saturation:saturation brightness:brightness alpha:alpha];
+}
+
+static inline UIColor *MDCColorLighten(UIColor *color, CGFloat percent) {
+  return MDCColorDarken(color, -percent);
+}
+
+@interface ChipsNoopTest : XCTestCase
```
Comment (yarneo) [inline components/Chips/tests/unit/ChipsTests.m]: Did you mean to call this class `ChipsTests` instead?

File: components/Chips/tests/unit/ChipsTests.m [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+@end
+
+@implementation ChipsNoopTest
```
Comment (yarneo) [inline components/Chips/tests/unit/ChipsTests.m]: Did you mean to call this class `ChipsTests` instead?

**Review by yarneo — DISMISSED** [review]

File: components/Chips/tests/unit/ChipsTests.m (hunk 1 above)
Comment (romoore) [reply components/Chips/tests/unit/ChipsTests.m]: Yep, good catch. I renamed the file and forgot to rename the class. :P

**Comment by yarneo** [comment]
One open question I wasn't sure about and wanted to know if you had thought about it. Do you think we might need to update the chipView font in `prepareForReuse` in the `MDCChipCollectionViewCell`?

**Comment by romoore** [comment]
If we touch the font in `prepareForReuse` it will disable UIAppearance. I don't think we should be doing that within the component.

**Review by yarneo — APPROVED** [review]

**Pull request merged**